staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -193,6 +197,7 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		openAPIConfig:              c.GenericConfig.OpenAPIConfig,
 		egressSelector:             c.GenericConfig.EgressSelector,
 		proxyCurrentCertKeyContent: func() (bytes []byte, bytes2 []byte) { return nil, nil },
+		CrdApiRegistrationDoneCh:   make(chan struct{}),
```

Comment:
I still have doubts about this is the correct way to order the controllers(autoregistration&apiserviceregistration).